ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -205,13 +180,14 @@ function getUrlParamsForEntity(selectEntity: {
 export type NetworkGraphProps = {
     detailType?: UrlDetailTypeValue;
     detailId?: string;
+    closeSidebar: () => void;
+    onSelectNode: (type, id) => void;
 };
 
 export type TopologyComponentProps = NetworkGraphProps;
 
-const TopologyComponent = ({ detailType, detailId }: TopologyComponentProps) => {
-    const selectedEntity = detailId && findEntityById(model, detailId, detailType);
-    const history = useHistory();
+const TopologyComponent = ({ detailId, closeSidebar, onSelectNode }: TopologyComponentProps) => {
```

Comment:
@alwayshooin @vjwilson  I'm actually not too fond of the prefix `detail`. I think that it's a bit confusing. What do you all think about `nodeType` and `nodeId` instead? Or if we eventually do want to make edges selectable, then we can do `objectType` and `objectId` or `elementType` and `elementId`. Open to other suggestions too

Reply:
i think `nodeType` and `nodeId` works! btw, you can select a node from the graph via the controller by saying `controller.getNodeById`. not sure if that simplifies things or if you've explored that but just wanted to let u know in case u werent aware

Reply:
@alwayshooin oh wow I didn't know that. Maybe we can try to use that function instead of making a custom one

Reply:
@vjwilson I think `node` or `object` is pretty much what we've called the selectable parts of our network graph regardless of whatever library we've used, so I don't think the terminology is specific to this library. I think in any graph, it's pretty common to have `nodes` and `edges`. If we ever decide to ditch the graph aspect of this view and go with something else, sure, then maybe saying `nodeType` and `nodeId` might not work anymore. I think for the foreseeable future it does make sense, but if it's a big issue we can also just do `type` and `id`